module/zfs/zio.c

```diff
@@ -2704,6 +2707,7 @@ zio_vdev_io_start(zio_t *zio)
 		/*
 		 * The mirror_ops handle multiple DVAs in a single BP.
 		 */
+		zio->io_delay = gethrtime();
```

Comment:
I think both this line and the`zio->io_delay = gethrtime()` should be dropped.  In the first case it's redundant and in the mirror case the mirror-type vdev will submit new zios to the leaves.  We just want to count physical zio's at the leaves to avoid double counting.  But please double check this.